cartography/intel/aws/iam.py

```diff
@@ -76,10 +80,10 @@ def get_group_managed_policy_data(boto3_session: boto3.session.Session, group_li
     policies = {}
     for group in group_list:
         name = group["GroupName"]
-        arn = group["Arn"]
+        group_arn = group["Arn"]
         resource_group = resource_client.Group(name)
-        policies[arn] = {
-            p.policy_name: p.default_version.document["Statement"]
+        policies[group_arn] = {
+            p.arn: p.default_version.document["Statement"]
```

Comment:
We are using the arn as the key in the dict returned by `get_group_managed_policy_data()`, instead of the policy name. This lets us dedupe managed policies.